lib/tests/suites/argonaut_sprint_1_test.rb

```diff
@@ -65,13 +70,13 @@ def define_metadata(method)
           validates resource: "Patient", methods: ["read", "search"]
         }
 
-        @id = @client.read_feed(@rc).resource.entry.first.resource.xmlId
+        @patient_id ||= @client.read_feed(@rc).resource.entry.first.resource.xmlId
```

Comment:
do we need a rescue on this? Won't this blow up on us if they don't support getting all patients?